packages/airnode-node/src/evm/handlers/initialize-provider.ts

```diff
@@ -142,32 +194,58 @@ export async function initializeProvider(
   // STEP 6: Fetch authorizations and transaction counts
   // =================================================================
   // NOTE: None of these promises can fail otherwise Promise.all will reject
-  const authAndTxCountPromises: readonly ParallelPromise[] = [
-    fetchSameChainAuthorizations(state5),
-    fetchTransactionCounts(state5),
-    fetchCrossChainAuthorizations(state5),
-  ];
+
+  // If all authorizers arrays are empty then all requests are authorized
+  const allAuthorizersEmpty = Object.values(state5.settings.authorizers).every((arr) => isEmpty(arr));
```

Comment:
This should deserve an INFO log I think.

Reply:
Good idea, I added the log to the if/else block below